src/SCSS/home.scss

```diff
@@ -52,12 +52,25 @@
   display: flex;
   justify-content: center;
   gap: 3rem;
-  margin-top: 4rem;
+  margin-top: 0;
 }
 
 img {
   width: 360px;
   height: 300px;
+  margin-left: 1rem;
+}
+
+h5 {
+  text-align: center;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  margin-left: 3rem;
+}
```

Comment:
Kindly adjust the car properties styling to fit with the original design 😉. Points to notice:
- The color of the text on the original design is lighter.
- The text should be aligned with the other elements.
- Before the model name, you can add Tesla (e.g. Tesla X Plaid).
- The social icons should look alike.

| Behance design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198748797-595b7666-abb0-40ed-bbfb-5f296a599a0d.png)  | ![image](https://user-images.githubusercontent.com/42799579/198748731-f8f3ec63-0091-43c2-a3b7-7839608d6d87.png)  |